active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,503 @@
+# Background
+> This spec corresponds to [issue 913](https://github.com/microsoft/microsoft-ui-xaml/issues/913) on the WinUI repo.
+
+Users should be informed about essential status changes that occur on an app level.
+These status changes affect the app as a whole and can be either critical or informational.
+Critical status changes like lost internet connectivity are directly impactful to app functionality while informational status changes like an update has completed and been applied are indirectly impactful to app functionality.
+These notifications and corresponding information should be presented in a consistent, predictable, and relevant way to the user depending on the specific scenario.
+
+For example the "Error while saving" message in this sample app:
+
+![initial example](images/initial-example.jpg)
+
+Currently, [TeachingTip](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.TeachingTip), 
+[ContentDialog](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ContentDialog), 
+and customizations of other [Flyouts](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Flyout) and dialogs exist as options to show these notifications but these controls were not specifically designed to handle app-wide status change notifications. 
+Due to their visual layouts, inherent intrusiveness, or available features they are not sufficient for displaying notifications at an app-wide level.
+
+This spec introduces an InfoBar WinUI (Xaml) control that an app can use for these kinds of messages.
+
+# InfoBar Class
+
+Represents an informational control that is visible but does not interfere with the user. 
+
+## Is this the right control?
+Use an InfoBar control when a user should be informed of, acknowledge, or take action on a message. By default the notification will remain in the content area until dismissed by the user but will not necessarily break user flow.
+
+An InfoBar will take up space in your layout and behave like any other child elements. It will not cover up other content or float on top of it.
+
+Do not use an InfoBar control to confirm or respond directly to a user action, for time-sensitive alerts, or for non-essential messages.
+
+## Remarks
+Use an InfoBar that is dismissed by the user or when the status is resolved for scenarios that **directly** impact app perception or experience
+
+- Internet connectivity lost
```

Comment:
nit: list seems to be disconnected from the sentence above. Perhaps add a line like. 'Here are some examples'